src/components/AddTodoForm.js

```diff
@@ -1,31 +1,31 @@
 
-import React,{useState} from 'react';
+import React, { useState } from 'react';
 import InputWithLabel from './InputWithLabel';
 
-const AddTodoForm = ({onAddTodo}) => {
-   const[todoTitle, setTodoTitle] = useState('')
-   const handleTitleChange = (event) =>{
-      const newTodoTitle = event.target.value
-      setTodoTitle(newTodoTitle)
-   }
-   const handleAddTodo = (event) =>{
-       event.preventDefault()
-       //let todoTitle = event.target.value
-       //console.log(todoTitle)
+const AddTodoForm = ({ onAddTodo }) => {
+  const [todoTitle, setTodoTitle] = useState('')
+  const handleTitleChange = (event) => {
+    const newTodoTitle = event.target.value
+    setTodoTitle(newTodoTitle)
+  }
+  const handleAddTodo = (event) => {
+    event.preventDefault()
+    //let todoTitle = event.target.value
+    //console.log(todoTitle)
 
-       onAddTodo({ title:todoTitle, id: Date.now() } )
-       setTodoTitle('')
-       //event.target.reset()    
-   }
+    onAddTodo({ fields: {Title:todoTitle} })
+    setTodoTitle('')
+    //event.target.reset()    
+  }
 
-   return( 
+  return (
     <>
       <form onSubmit={handleAddTodo} >
-        <InputWithLabel todoTitle={todoTitle} handleTitleChange= {handleTitleChange}>Title: 
-       </InputWithLabel>
-       &nbsp;
-       <button type="Submit">  Add</button>       
-      </form> 
+        <InputWithLabel todoTitle={todoTitle} handleTitleChange={handleTitleChange}>Title:
+        </InputWithLabel>
+        &nbsp;
+        <button type="Submit">  Add</button>
+      </form>
     </>
```

Comment:
There are a couple of minor issues here, like unnecessary spaces before the words in the `<button>` tags, and the unnecessary fragment. Fragments are only needed when we need to group several tags together. Here, the `<form>` tag wraps everything so we don't need the fragment.